wordle-web/components/hamburger-menu.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <v-card>
```

Comment:
You could also wrap this in a `div` so your button matches the look of the settings button. Or you could use `v-card flat >` to get rid of the drop shadow on it.